### PR TITLE
Add statistics for dashboard

### PR DESCRIPTION
**Pull request by mosuem**
Calculates some statistics, this could be used to make a first iteration of a dashboard.

Todo for a follow-up PR is to make a nice markdown summary of the generated statistics.

### CODE REVIEW

File: pkgs/compute_statistics/bin/compute_statistics.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:compute_statistics/compute_statistics.dart';
+import 'package:compute_statistics/statistics_type.dart';
+import 'package:dart_triage_updater/firebase_database.dart';
+
+Future<void> main(List<String> arguments) async {
```
Comment (devoncarew) [inline pkgs/compute_statistics/bin/compute_statistics.dart:8]: I think its the same either way, but this functionality could also be added to the existing cli tool in this repo as a separate command (convert that cli tool to supporting multiple commands).

File: pkgs/compute_statistics/lib/compute_statistics.dart [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    return diffList;
+  }
+
+  DateTime getFrom(int i) =>
+      DateTime(referenceDate.year, referenceDate.month - i);
+
+  DateTime getTo(int i) =>
+      DateTime(referenceDate.year, referenceDate.month - i + 1);
+
+  List<Issue> getPIssues(int i, List<Issue> openIssues) {
```
Comment (devoncarew) [inline pkgs/compute_statistics/lib/compute_statistics.dart:128]: What is a PIssue, PLabel, PPullRequest?

File: pkgs/compute_statistics/lib/compute_statistics.dart [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        .where((pr) =>
+            pr.labels?.any((label) => isPLabel(priority, label)) ?? false)
+        .toList();
+  }
+
+  Future<List<T>> getOpen<S, T>(UpdateType<S, T> type) async =>
+      await ref.getAllWith(type, orderBy: 'state', equalTo: 'open');
+
+  bool isPLabel(int priority, IssueLabel label) {
+    //TODO: Account for different types of priority labels
```
Comment (devoncarew) [inline pkgs/compute_statistics/lib/compute_statistics.dart:145]: We should very soon just have one kind :)

**Review by devoncarew — APPROVED** [review]

File: pkgs/compute_statistics/bin/compute_statistics.dart (hunk 1 above)
Comment (mosuem) [reply pkgs/compute_statistics/bin/compute_statistics.dart:8]: It feels a bit cleaner this way, but merging the two does make sense.

**Pull request merged**